package common.scalable;

message PCMonthlyAwardGot
{
    repeated int32 cumulative = 1;
}
